### PR TITLE
`jj resolve --list` and `jj status`: make conflict output colorful + clippy

### PR DESCRIPTION
**Pull request by ilyagr**
`jj status` now looks like this:

image

# Checklist

If applicable:
- (not planned) I have updated `CHANGELOG.md`
- (not planned) I have updated the documentation (README.md, docs/, demos/)
- [x] I have added tests to cover my changes

### CODE REVIEW

**Comment by ilyagr** [comment]
I added a commit to do this to `jj status` as well. It doesn't seem to have the right test, but here's a picture: 
image

File: src/commands.rs [hunk 1]
```diff
@@ -1539,9 +1539,7 @@ fn cmd_status(
             formatter.with_label("conflict", |formatter| {
                 writeln!(formatter, "There are unresolved conflicts at these paths:")
             })?;
-            for (path, _) in conflicts {
-                writeln!(formatter, "{}", &workspace_command.format_file_path(&path))?;
-            }
+            print_conflicted_paths(&conflicts, &tree, formatter, &workspace_command)?
```
Comment (martinvonz) [inline src/commands.rs:1542]: It would be nice if the paths were indented just like the conflicted branches above are, but that can be fixed later.

**Review by martinvonz — APPROVED** [review]
Thanks

File: src/commands.rs (hunk 1 above)
Comment (ilyagr) [reply src/commands.rs:1542]: The simplest thing would be to have a `formatter.with_indent` function. I also imagine `jj log` has to do something even fancier to print the commit descriptions to the right of the graph.

**Pull request merged**